### ISSUE TITLE
Better header for mobile

### ISSUE THREAD

**Issue by iangilman** [pixfabrik/artywidget#35]
It's starting to fill up!

<img width="365" alt="image" src="https://user-images.githubusercontent.com/338989/32525924-a25ce420-c3db-11e7-8929-c670a613ef6e.png">

We probably need to hide some stuff in a menu.

**Comment by kittarlin** [comment]
Put at down carrot next to the username and hide the log out link under that.  Consider doing the same next to the Arty Widget name.  Maybe have the user name replaced with a generated icon/avatar for small screens?

cheers,
christina

**Comment by iangilman** [comment]
Yeah, I think having a menu on the left (for home, about, play, and add artwork) and another on the right (for profile and logout) makes sense for mobile.

I think we can stick with the username if we do that, though of course eventually adding avatars would be nice.
